### DISCUSSION TITLE
看了文档，跑了下demo，有些问题，来请教一下

### DISCUSSION THREAD

**Discussion by feng-fu** [empjs/emp#207] [Q&A]
1. remotes中的引用，初始化项目后默认写为component@http://localhost:8000/emp.js ，这个在开发完后上线时，需要自己手动改，还是可以通过环境变量来控制，有没有对于的示例呢

2. 我现在有两个应用，一个是base应用，一个remote应用，在base应用中引用了remote引用，如果remote中有修改，是不是需要发布remote和base两个应用。

**Comment by ckken** [comment]
只需要发布 remote
